rizzino_a09b.js

```diff
@@ -0,0 +1,51 @@
+/* Julie Rizzino
+rizzino_a09b.js
+INFO 2124
+Thoendel
+2/12/20 */
+
+document = "rizzino_a09b.html";
+//define the document for the js file
+
+var pCount = document.getElementsByTagName('p');
+//get the p tag element from the html document and label it pCount
+let msgP = `The total number of p tags in this document is: ${pCount.length}.<br>`;
+//message for output in the html file to display the length of the var pCount
+console.log(msgP);
+//output message to the console when accessing html file
+
+var divCount = document.getElementsByTagName('div');
+//get the div tag element from the html document and label it divCount
+let msgDiv = `The total number of div tags in this document is: ${divCount.length}.<br>`;
+//message for output in the html file to display the length of the var divCount
+console.log(msgDiv);
+//output message to the console when accessing html file
+
+var paraCount = document.getElementsByClassName('paragraph');
+//get the paragraph class element from the html document and label it paraCount
+let msgPara = `The total number of paragraph class elements in this document is: ${paraCount.length}.<br>`
+//message for output in the html file to display the length of the var paraCount
+console.log(msgPara);
+//output message to the console when accessing html file
+
+var textCount = document.getElementsByClassName('block-text');
+//get the blocl-text class element from the html document and label it textCount
+let msgText = `The total number of block-text class elements in this document is: ${textCount.length}.<br>`
+//message for output in the html file to display the length of the var textCount
+console.log(msgText);
+//output message to the console when accessing html file
+
+var dispId = document.getElementById('b001');
```

Comment:
This works - but if you then chain on innerhtml - you would get a little better display.

Nice job!